static/app/views/integrationPipeline/awsLambdaFunctionSelect.tsx

```diff
@@ -37,7 +37,7 @@ const getLabel = (func: LambdaFunction) => func.FunctionName;
 export default class AwsLambdaFunctionSelect extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
-    makeObservable(this);
+    makeObservable(this, {allStatesToggled: computed});
```

Comment:
This was my attempt to fix the error, by being explicit about what's observable 🤷